src/main/haskell/kore/src/Kore/AST/Common.hs

```diff
@@ -103,20 +103,20 @@ instance Hashable (Variable level)
 
 instance NFData (Variable level)
 
-{- | @Concrete level@ is a variable occuring in a concrete pattern.
+{- | @ConcreteVariable level@ is a variable occuring in a concrete pattern.
 
-    Concrete patterns do not contain variables, so this is an uninhabited type
+    ConcreteVariable patterns do not contain variables, so this is an uninhabited type
     (it has no constructors).
 
     See also: 'Data.Void.Void'
 
  -}
-data Concrete level
+data ConcreteVariable level
```

Comment:
Is this necessary? I think including `Variable` in the name is deeply misleading since it is uninhabited.